styles.css

```
@@ -0,0 +1,322 @@
+}
+
+blockquote,
+q {
+  quotes: none;
+}
+
+blockquote::before,
+blockquote::after,
+q::before,
+q::after {
+  content: '';
+  content: none;
+}
+
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+.top-nav {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  margin-top: 15px;
+}
+
+.top-nav1 {
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+  padding: 0;
+}
+
+.top-nav1 a,
+p {
+  font-size: 14px;
+  margin-right: 10px;
+}
+
+.top-nav-button {
+  background-color: rgb(51, 102, 255);
+  border-radius: 5px;
+  border-width: 1px;
+  height: 100%;
+  display: flex;
+  flex-direction: row;
+  color: white;
+  align-items: center;
+}
+
+.nav-button {
+  display: flex;
+  margin: 10px;
+  flex-direction: row;
+}
+
+.nav-button p {
+  margin: 0;
+  padding-left: 10px;
+}
+
+.fa-lock {
+  margin: 0;
+  color: white;
+}
+
+.main-logo {
+  margin: 10px;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: white;
+  width: 380px;
+  height: 600px;
+  margin: 10px 10px 0 10px;
+  border: solid;
+  border-bottom: transparent;
+  border-width: 1px;
+  border-color: darkgray;
+  box-sizing: border-box;
+}
+
+form h3 {
+  margin: 5%;
+}
+
+.form-head1 {
+  margin: 2%;
+  width: 85%;
+}
+
+.form-head2 {
+  margin: 2%;
+}
+
+.align-form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+}
+
+.align-form button {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 85%;
+  height: 40px;
+}
+
+.form-button {
+  display: flex;
+  justify-content: space-around;
+  margin-top: 15px;
+  margin-bottom: 15px;
+  width: 100%;
+  background-color: rgb(51, 102, 255);
+  border: transparent;
+}
+
+.form-button p {
+  color: white;
+}
+
+.label-section {
+  display: flex;
+  flex-direction: column;
+  text-align: start;
+  width: 85%;
+  margin-top: 15px;
+  margin-bottom: 15px;
+}
+
+.label-section1 {
+  margin-top: 15px;
+  margin-bottom: 15px;
+}
+
+.label-section2 {
+  margin-top: 15px;
+  margin-bottom: 15px;
+}
+
+.label-section3 {
+  margin-top: 15px;
+  margin-bottom: 15px;
+}
+
+.label-section-p {
+  font-size: 10px;
+}
+
+.form-bottom {
+  display: flex;
+  background-color: rgb(226, 226, 226);
+  border: solid;
+  border-width: 1px;
+  border-color: darkgray;
+  width: 380px;
+  border-top: transparent;
+  padding: 1.8em;
+  font-size: 0.7em;
+}
+
+.form-bottom p {
+  display: flex;
+  margin-left: 20px;
+  font-size: 10px;
+}
+
+.form-button1 {
+  font-style: italic;
+  width: 85%;
+  margin-top: 5px;
+  margin-bottom: 15px;
+}
+
+.form-button1 p {
+  font-size: 10;
+  margin-top: 5px;
+}
+
+footer {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin: 15px;
+}
+
+.footer-div {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  margin: 8px;
+}
+
+.footer-div p {
+  font-size: 10px;
+  margin-left: 5px;
+  margin-right: 5px;
+}
```

Your footer needs a little tweak in order to look like the actual site's. 
```suggestion
}footer {
    display: flex;
    flex-direction: column;
    width: 100%;
    margin-top: 24px;
}

.footer-div {
    display: flex;
    flex-direction: row;
    justify-content: center;
    /* margin: 8px; */
    margin-bottom: 5px;
}

.footer-div p {
    font-size: 12px;
    /* margin-left: 5px; */
    /* margin-right: 5px; */
    line-height: 0;
    color: rgb(80, 80, 80);
    font-weight: 500;
    font-family: sans-serif;
}

```